assets/cex/bingx.json

```diff
@@ -23,6 +23,14 @@
             "tags": [],
             "submittedBy": "shuva10v",
             "submissionTimestamp": "2025-02-18T00:00:00Z"
+         },
+  {
+            "address": "EQA7GywcoAEBQX2cLZoyK0UggR6uizFbvfjpuqbMemN30",
```

Comment:
This is not the address that you mentioned in your PR description. This addr does not exist. 

This is the one you've mentioned in PR (the valid address)
`EQA7Gy_WcoAeBQX2cLZoyK0UggR6uJi2F_bVfjpupbMemN3O`